jmix-gridexport/gridexport-flowui/src/main/java/io/jmix/gridexportflowui/action/ExportAction.java

```diff
@@ -95,6 +104,65 @@ public void setGridExportProperties(GridExportProperties gridExportProperties) {
         this.availableExportModes = gridExportProperties.getDefaultExportModes().stream()
                 .map(ExportMode::valueOf)
                 .toList();
+        this.columnExportFilter = ColumnExportFilter.valueOf(gridExportProperties.getDefaultColumnExportFilter());
+    }
+
+    /**
+     * Sets the {@link ColumnExportFilter} that is used to filter columns to export. This is a simple, predefined
+     * alternative to {@link #setColumnExportPredicate(Predicate)}. Has the lowest filtering priority.
+     * <p>
+     * The default value depends on {@link GridExportProperties#getDefaultColumnExportFilter()}.
+     *
+     * @param columnExportFilter column export filter to set
+     */
+    public void setColumnExportFilter(ColumnExportFilter columnExportFilter) {
```

Comment:
From users point of view, they set column export mode either **all columns** or **visible**. The fact that this enum stores filter predicates is hidden from users. Just read this: _set column export filter all rows_. It sounds like we need to filter all columns, but in fact, it's opposite. Let's rename Enum, app property and method.